asyncscope.md

```diff
@@ -435,18 +435,19 @@ return when_all(s.on_empty(), std::move(snd));
 template <sender S> @_nest-sender_@<S> nest(S&& s);
 ```
 
-Returns a `@_nest-sender_@` that extends the lifetime of the `async_scope` that produced it to include the lifetime of the `@_nest-sender_@` object and, if started, the lifetime of the given sender operation.
+Returns a `@_nest-sender_@` that, when started, extends the lifetime of the `async_scope` that produced it to include the lifetime of the `@_nest-sender_@` object and the lifetime of the given sender operation.
 
 A call to `nest()` does not start the given sender.
 A call to `nest()` is not expected to incur allocations.
 
-Connecting and starting the sender returned from `nest()` will connect and start the input sender.
+The sender returned by a call to `nest()` holds a reference to the `async_scope`.
+Connecting and starting the sender returned from `nest()` will connect and start the input sender and will extend the `async_scope`'s lifetime to include the `@_nest-sender_@` and given sender operation.
 
 Similar to `spawn_future()`, `nest()` doesn't constrain the input sender to any specific shape.
 Any type of sender is accepted.
 
-The returned sender prevents the scope from ending.
-It is safe to drop the returned sender without starting it, because the `async_scope` safely manages the lifetime of the running operations.
+Unlike `spawn_future()` the returned sender does not prevent the scope from ending.
+It is not safe to drop the returned sender without starting it.
```

Comment:
why is it not safe to drop the sender returned from nest()?

Reply:
I inverted the semantic here, where elsewhere I just dropped it. I think it's more accurate to say nothing. We gain no safety here, but we don't lose it either. That assumes that the nest sender does nothing beyond grabbing a reference to the scope and the code to do the ref counting.